docs/web-apps/automated-testing/selenium/selenium4.md

```diff
@@ -47,41 +50,40 @@ Below are features new to Selenium 4 that may impact your tests.
 </tr>
 <tr>
 <td>Chromium Edge Options methods (Se3 technically only supported Edge HTML)</td>
-<td>z </td>
+<td>?</td>
 </tr>
 <tr>
 <td>RemoteWebDriverBuilder (Java Only)</td>
-<td>In order to use browser specific-options, you have to cast it & augment it (access local webdriver methods). The .builder() method allows you to send more than url & browseroptions. Now you can do more like send browser.options, timeout. Before there were a lot of defaults & you didn’t have as many options for starting a driver. Create a hashmap instead of having mutable capabilities. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/RemoteWebDriverBuilderTest.java">Code example</a>.</td>
+<td>Prior to Selenium 4, in order to use browser specific-options, you had to cast it and augment it (access local webdriver methods). There were a lot of defaults and you didn’t have many options for starting a driver. With Selenium 4's new <code>.builder()</code> method, you can send <code>browser.options</code>, timeouts, and create a hashmap instead of having mutable capabilities. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/RemoteWebDriverBuilderTest.java">See a code example</a>.</td>
 </tr>
 <tr>
 <td>New Element Attribute and Property methods</td>
-<td>Two new methods have been added to more precisely get a given element attribute or a property. These are especially useful for performance and preciseness in Sauce Labs tests. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/AttributePropertyTest.java">Code example</a>.</td>
+<td>Two new methods have been added to more precisely get a given element attribute or a property. These are especially useful for performance and preciseness in Sauce Labs tests. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/AttributePropertyTest.java">See a code example</a>.</td>
 </tr>
 </table>
 
 
 ## Preparing Your Test Code
 
-As mentioned previously, Selenium 4 deprecates support for legacy JSON wire protocol and its static browser methods `DesiredCapabilities`. It will be replaced by W3C WebDriver-compliant protocol and the use of `browserOptions`. Using `browserOptions` simplifies the configuration needed to start a new session, allows you to set browser-specific settings (like headless in Chrome), and reduces the chances of browser misconfiguration.
+Because Selenium 4 deprecates support for legacy JWP (`DesiredCapabilities`) &#8212; replacing it with W3C WebDriver-compliant (`browserOptions`) &#8212; you'll need to prepare your test code accordingly. Using W3C `browserOptions`:
```

Comment:
```suggestion
Because Selenium 4 deprecates support for legacy JSON wire protocol (`DesiredCapabilities`) &#8212; replacing it with W3C WebDriver-compliant (`browserOptions`) &#8212; you'll need to prepare your test code accordingly. Using W3C `browserOptions`:
```